pkg/eval/builtin_fn_container_test.go

```diff
@@ -203,17 +235,20 @@ func TestOrder(t *testing.T) {
 		That("put 10 1 5 2 | order").Puts("1", "10", "2", "5"),
 
 		// Ordering numbers
-		// Only small integers
+		// Only small integers normalized to the "num" type.
```

Comment:
"Normalization" has a specific meaning when it comes to number types. To clarify that these are values of number types instead of strings, say "typed numbers".